### DISCUSSION TITLE
Maneuver at the end of the edge

### DISCUSSION THREAD

**Discussion by Gtx513** [Courseplay/Courseplay_FS22#2956] [General]
Hello everyone, I had a quick question please: is there a solution for the tractor to use a more realistic maneuver at the end of the plot
 , than making a loop during the maneuver? Thanks you so much.

**Comment by Tensuko** [comment]
I dont know what you mean with plot and what a more realistic turn is.

**Comment by Tensuko** [comment]
Closed due to inactivity.
